File: Poin B.php/Hardware.php
<?php

//Include File Parent
include "Product.php";


class Hardware extends Product{
	//Deklarasi Atribut
    private $brand;
    private $model;


	//Constructor
	function __construct() {
	}

	//Get Set Methods
	function getBrand() {
		return $this->brand;
	}
	
	function setBrand($brand){
		$this->brand = $brand;
	}

	function getModel() {
		return $this->model;
	}

	function setModel($model){
		$this->model = $model;
	}

	//Output Methods
	function output_hardware(){
		$this->output_product();
		echo "Brand					: ". $this->getBrand()."<br/>";
		echo "Model					: ". $this->getModel()."<br/>";


	}


	//Destructor
	function __destruct()
    {
        
    }
}
?>

File: Poin B.py/Hardware.py
from itertools import product


#import kelas parent
from Product import Product


class Hardware(Product) :
    #inisialisasi atribut
    __Brand = ""
    __Model = ""
    
    #constructor
    def __init__(self, brand= "", model = "") :
        self.__Brand = brand
        self.__Model = model
        
    #getset methods
    def setBrand(self, brand = "") :
        self.__Brand = brand
        
    def getBrand(self) :
        return self.__Brand
    
    def setModel(self, model = "") :
        self.__Model = model
        
    def getModel(self) :
        return self.__Model
    
    #output method
    def outputHardware(self):
        self.outputProduct()
        print("Brand                         : "+ str(self.getBrand()))
        print("Model                         : "+ str(self.getModel()))
        
    
        
    